Makefile

```diff
@@ -11,7 +11,9 @@ build  :; forge build
 test   :; forge test -vv
 test-contract :; forge test --match-contract ${filter} -vvv
 
-test-usdt-v3-ethereum :; forge test -vvv --match-contract AaveV3EthUSDTPayloadTest
+
+deploy-ledger :; forge script ${contract} --rpc-url ${chain} $(if ${dry},--sender 0x25F2226B597E8F9514B3F68F00f494cF4f286491 -vvvv,--broadcast --ledger --mnemonics foo --mnemonic-indexes ${MNEMONIC_INDEX} --sender ${LEDGER_SENDER} --verify -vvvv)
```

Comment:
Why the sender is hardcoded?

Reply:
it's the ecosystem reserve, so the dryrun will succeed on mainnet proposal creation.
Could also put `${LEDGER_SENDER}` here, but wasn't sure what to put on pk... wdyt?